miscellaneous/build_templates/analysis.yml

```diff
@@ -13,6 +13,12 @@ jobs:
       name: 00-OSIManaged-Test
       demands: COMPUTERNAME -equals $(buildAgent)
     steps:
+      # Remove ".placeholder" from filenames
+      -script: |
+          for filename in *; do [ -f "$filename" ] || continue; mv "$filename" "${filename//.placeholder/}"; done
```

Comment:
I'm not 100% sure but I think you can break each of these lines out just to make it a little more readable. Otherwise I think this looks good. For example:

```bash
for file name in *
do [ -f "$filename" ] || continue
...
```